[W-10890447] API Manager - create API wizard UI updates (#452)
mulesoft/docs-api-manager: modules/ROOT/pages/promote-api-task.adoc

A tight integration of Anypoint Platform components extends the use of environments to API management. In this procedure, you change the environment of an API.

> NOTE: You must have Manage APIs Configuration permission to promote an API. In addition you must have Access Management permission to set permissions for viewing and managing an entity. You can only promote an entity (Policies, Alerts, etc.) if you have permission to view the source and permission to manage the destination over each entity. Typical view permissions are View API Configuration and View Policies as explained in xref:environment-permission-task.adoc[To Grant Permission to Access Environments]. Set the permissions as needed before proceeding.

Reviewer: ```suggestion
You must have `Manage APIs Configuration` permission to promote an API and `Access Management` permission to set permissions for viewing and managing an entity. You can only promote an entity (policies, alerts, and so forth) if you have permission to view the source and permission to manage the destination over each entity. Typical view permissions are `View API Configuration` and `View Policies`, as explained in xref:environment-permission-task.adoc[To Grant Permission to Access Environments]. Set the permissions as needed before proceeding.
```

Reviewer: What does "manage the destination over each entity" mean?

Reviewer: This is a lot of text to be in a note. I don't think it needs to be in a note.

Author: Agreed, will update

. Select *API Manager > API Administration > Add API > Promote API from environment*.

> . For *Source Environment*, select the environment of the API you want to promote.

Reviewer: ```suggestion
. For *Source Environment*, select the environment of the API to promote.
```

Author: will update

> . For *API*, enter the name of and then select an API from the APIs in the source environment.

Reviewer: ```suggestion
. For *API*, enter the name of the API and then select it API from the APIs in the source environment.
```

Reviewer: Are you selecting the same API that you enter or a different one? If it's a different one: "and then select another API..."

Author: it is the same - will fix
